fhir/resource_structure_dev/research_study.json

```diff
@@ -0,0 +1,111 @@
+{
+  "resourceType" : "ResearchStudy",
+  "id" : "example-study-ccpuk",
+  "url" : "",
+  "name" : "CCPUK",
+  "title" : "ISARIC/WHO Clinical Characterisation Protocol UK",
+  "label" : [{
+    "type": {
+      "text": "Study Code"
+    },
+    "value": "CVCCPUK"
+  }],
+  "status" : "active",
+  "primaryPurposeType" : {
+    "coding" : [{
+      "system" : "http://terminology.hl7.org/CodeSystem/research-study-prim-purp-type",
+      "code" : "diagnostic",
+      "display" : "Diagnostic"
+    }]
+  },
+  "phase" : {
+    "coding" : [{
+      "system" : "http://terminology.hl7.org/CodeSystem/research-study-phase",
+      "code" : "phase-1",
+      "display" : "Phase 1"
+    }]
```

Comment:
the data element studyDesign can be used to give high-level classification on the type of study (observation/intervention, randomisation (and type) or not, multisite etc). may need more than one per study